var/connect/Sailthru_Email.xml

```diff
@@ -18,9 +18,11 @@
 </description>
    <license>Apache License 2.0</license>
    <license_uri>http://www.apache.org/licenses/LICENSE-2.0</license_uri>
-   <version>2.0.0</version>
-   <stability>stable</stability>
-   <notes>Large update adding many new features, more granular configuration, auto-tagging, and resolving issues with API connectivity, For more information, visit our documentation at https://getstarted.sailthru.com/integrations/magento/magento-1-extension-v2/.</notes>
+   <version>2.1.0-beta</version>
```

Comment:
Same as previous question about labels. Will we update this when moving to master and prior to releasing.

Reply:
Yup